resources/checks/A004_cluster_info.sh

```diff
@@ -1,6 +1,8 @@
-# Collect pg_settings artifact
+# Collect pg cluster info
 #dbg "PSQL_CONN_OPTIONS: ${PSQL_CONN_OPTIONS}"
-pgver=$(psql ${PSQL_CONN_OPTIONS}  -c "SHOW server_version" -t -A)
+
+pgver=$(ssh ${HOST} "${_PSQL} -c \"SHOW server_version\"")
```

Comment:
it's a bit easer to get `show server_version_num` here and the make interger comparison, w/o any regexp-replacement